bundles/org.openhab.binding.harmonyhub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,6 +7,7 @@
 	<bridge-type id="hub">
 		<label>Harmony Hub</label>
 		<description>A Logitech Harmony Hub</description>
+		<semantic-equipment-tag>AudioVisual</semantic-equipment-tag>
```

Comment:
```suggestion
		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

like in #18541.

Reply:
This one is a complex case; the "hub" has both the functions of a Bridge (Internet etc.) but also of an A/V control device; let me think about this..

Reply:
@jlaur for avoidance of confusion I am removing this file from this PR so #18541 is the only one to discuss.